src/aims_ui/api_interaction.py

```diff
@@ -7,12 +7,12 @@
 
 def get_params(all_user_input):
   """Return a list of aparameters formatted for API header, from class list of inputs"""
-  params = 'verbose=True&&'
+  params = 'verbose=True&'
   for param in all_user_input.keys():
     # Do not add epoch for testing
     if (param != 'epoch') and (os.getenv('FLASK_ENV') == 'development'):
       if str(all_user_input.get(param)) != '':
-        params = params + str(param) +'='+ str(all_user_input.get(param)).replace('%','') + '&&'
+        params = params + str(param) +'='+ str(all_user_input.get(param)).replace('%','') + '&'
```

Comment:
When building the query string, it's probably best to use an actual URL escaping function rather than replacements. You probably want the `quote_plus` function from https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote_plus, ala:
```python
params = ['verbose=True']
for param, value in all_user_input.items():
  if not str(value):
    continue
  if (os.getenv('FLASK_ENV') == 'development') and (param == 'epoch'):
    # do not add epoch for testing
    continue
  params.append(urllib.parse.quote_plus(str(param) + '=' + str(value)))
return '&'.join(params)
```

Reply:
Adjusted to work